scaling-succotash.yaml

```
@@ -5,6 +5,11 @@ git:
 localhost:
   type: A
   value: 127.0.0.1
+pretzels:
```

Apparently the junk data wasn't a problem. Maybe list entries out-of-order?

```suggestion
candyland:
```